src/engine/StatusManager.java

```diff
@@ -64,4 +59,13 @@ private void saveStatus() {
             logger.warning("Failed to save status.");
         }
     }
+
+    public void resetDefaultStatus() {
+        try {
+            status = FileManager.getInstance().loadShipStatus();
+        } catch (IOException e) {
+            logger.warning("Failed to load status. Using default values.");
+            status = new ShipStatus(750, -6, 2, 100);
```

Comment:
```suggestion
            status = new ShipStatus(750, 6, 2, 100);
```
Default bulletSpeed가 음수로 되어 있어 수정이 필요합니다.